common/transaction-log.ts

```diff
@@ -0,0 +1,21 @@
+export type DocumentKind =
+  | 'txn'
+  | 'group'
+  | 'user'
+  | 'contract'
+  | 'contractBet'
+  | 'contractComment'
+
+export type WriteKind = 'create' | 'update' | 'delete'
+
+export type WriteDocument = { [k: string]: any }
+
+export type TLEntry<T extends WriteDocument = WriteDocument> = {
+  eventId: string
+  docKind: DocumentKind
```

Comment:
Technically they are already not 1:1 with collections. Bets come from many independent Firestore collections. I am also not sure about the future. In general, the important thing here is the type of the document, not where it came from, because the type is mostly what is going to make me want to make a separate SQL table for it.